source/includes/administration/_discounts.md

```diff
@@ -64,9 +64,67 @@ Attributes | &nbsp;
 `cutoffDate`<br/>*date* | The date on which the discount will no longer be available to customers who have not already received it. If not provided, the discount will always be available after the start date.
 `status`<br/>*enum* | The status of the discount. Possible values are: UPCOMING, CURRENT, ENDED.
 
+<!-------------------- GET DISCOUNT -------------------->
+### Get discount
 
-<!-------------------- CREATE DISCOUNTS -------------------->
-### Create discounts
+`GET /applied_pricings/:applied_pricing_id/discounts/:id`
+
+Retrieves a discount's details.
+
+```shell
+# Retrieve applied pricing list
+curl "https://cloudmc_endpoint/rest/applied_pricings/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts/08695c90-3e41-4be8-a3a3-df964620ba62" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "applyToNewCustomersOnly": true,
+      "discountedProducts": {},
+      "durationMonths": 22354,
+      "type": "PERCENTAGE",
+      "packageDiscount": 20.0,
+      "discountScope": "ALL_PRODUCTS",
+      "isDeactivated": true,
+      "discountedCategories": {},
+      "name": {
+        "en": "Summer Discount",
+        "fr": "Réduction Estival"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
```

Comment:
i think this should match the one in the url

Reply:
8a8df4722c9c5598dfe5b0432eedf20acd1ba951